packages/ace-core/src/core/get-all-rules.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const axe = require('axe-core');
+const mustache = require('mustache');
+const fs = require('fs');
+
+const Rules = require('../checker/rules');
+
+const PATH_TO_RULES_TEMPLATE = `${__dirname}/rules.mustache`;
+
+function mapUrlsToKB(axerules) {
+  // TODO: axe helping urls should be map to daisy knowledge database
+  // see kbMap in axe2ace.js
+  return axerules;
+}
+
+function getAllRules() {
+  const allRules = {};
+
+  // TODO: Does Ace really uss all rule from axe? Or the rule doc need to be filtered?
```

Comment:
Ace does filter out some rules, and add a couple custom rules (currently in [`ace-axe.js`](https://github.com/daisy/ace/blob/v0.8.0/packages/ace-core/src/scripts/ace-axe.js#L86-L190)).

This listing function should do the same, so we need to find a way to share the config.